packages/atomic/src/components/atomic-result/atomic-result-cell-desktop.pcss

```diff
@@ -62,7 +62,7 @@
       font-size: var(--font-size);
       --line-height: 1rem;
       line-height: var(--line-height);
-      margin-top: 1.75rem;
+      margin-top: 0.475rem;
```

Comment:
we could leverage a variable here too

Reply:
We could, but there's several margins like this for every section, I think if we do this it should be all or nothing, and I don't really see much value in adding variables for those. If people want different margins, that's already the point of the density setting.